torchinductor/compile_fx.py

```diff
@@ -71,7 +71,7 @@ def compile_fx_inner(
     num_fixed=0,
     is_backward=False,
 ):
-    log.info("Compiling %s graph", "BACKWARDS" if is_backward else "FORWARDS")
+    log.info("Inductor compiling %s graph", "BACKWARDS" if is_backward else "FORWARDS")
```

Comment:
For each graph we are trying to compile, let's give each of them an ID and print the ID as well. For example:
- "Inductor compiling graph 1 FORWARDS"
- "Inductor compiling graph 1 BACKWARDS"
- "Inductor compiling graph 2 FORWARDS"
- "Inductor compiling graph 2 BACKWARDS"
etc.